functions/Set-DbaAgentJob.ps1

```diff
@@ -0,0 +1,444 @@
+function Set-DbaAgentJob {
+    <#
+.SYNOPSIS 
+Set-DbaAgentJob updates a job.
+
+.DESCRIPTION
+Set-DbaAgentJob updates a job in the SQL Server Agent with parameters supplied.
+
+.PARAMETER SqlInstance
+SQL Server instance. You must have sysadmin access and server version must be SQL Server version 2000 or greater.
+
+.PARAMETER SqlCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. To use:
+$scred = Get-Credential, then pass $scred object to the -SqlCredential parameter. 
+To connect as a different Windows user, run PowerShell as that user.
+
+.PARAMETER JobName
+The name of the job. Can be null if the the job id is being used.
```

Comment:
Hey Sander, all of our other commands use just -Database or -Job, etc. Please specify just -Job here